sopel/modules/instagram.py

```diff
@@ -134,14 +142,18 @@ def parse_insta_json(json):
         parts.append('%sx%s' % (iwidth, iheight))
 
     # Likes
-    ilikes = needed.get('edge_media_preview_like', {}).get('count')
-    if ilikes:
-        parts.append('Likes: {:,}'.format(ilikes))
+    ilikes = str(needed.get('edge_media_preview_like', {}).get('count'))
+    if ilikes and ilikes.isdigit():
+        parts.append(
+            _format_counter(int(ilikes), 'No ♥ yet', '1 ♥', '{number} ♥s'))
```

Comment:
Why is that? It makes no sense to me.

Reply:
It's because grammatically, in English, 0 is treated as "plural" for _reasons_. I don't understand it either, but that's how the language works. 🤷‍♂️ 🤣